mula/scheduler/server/handlers/schedules.py

```diff
@@ -246,3 +255,36 @@ def patch(self, schedule_id: uuid.UUID, schedule: serializers.SchedulePatch) ->
             ) from exc
 
         return updated_schedule
+
+    def search(
+        self,
+        request: fastapi.Request,
+        offset: int = Query(0),
+        limit: int = Query(10),
```

Comment:
As far as I know fastapi handles arguments as query parameters by default. Body parameters are usually explicitly specified

```suggestion
        offset: int = 0,
        limit: int = 10,
```

Reply:
Cool, thought I had to explicitly specify it.